src/services/CompaniesService.js

```diff
@@ -0,0 +1,31 @@
+import http from './HttpService';
+
+const company = {
+    createCompany: async function () {
```

Comment:
I don't see the `createCompany` method being called anywhere - it should be called as a part of registration process :) we can use `${username}'s Company` as a default value